test/firebase-query.html

```diff
@@ -353,7 +359,8 @@
             return setFirebaseValue(
                 query.path + '/' + query.data[0].$key + '/val', object.val + 1);
           }).then(function() {
-            expect(domRepeat._instances[0].item).to.be.eql(query.data[0]);
+            var instances = domRepeat._instances || domRepeat.__instances;
```

Comment:
I just rechecked `domRepeat._instances` is undefined in Polymer 2 and this propertly seems to set `instance` to the value of `domRepeat.__instances`